docs/4_1_co-simulation_math.adoc

```diff
@@ -40,9 +40,7 @@ If multiple derivatives of a variable shall be retrieved, list the value referen
 * `orders` contains the orders of the respective <<derivative>> (1 means the first <<derivative>>, 2 means the second <<derivative>>, ..., 0 is not allowed).
 If multiple derivatives of a variable shall be retrieved, its value reference must occur multiple times in `valueReferences` aligned with the corresponding `orders` array.
 
-* `values` is a vector with the values of the <<derivative,`derivatives`>>.
-The order of the `values` elements is derived from a twofold serialization: the outer level corresponds to the combination of a value reference (e.g., `valueReferences[k]`) and order (e.g., `orders[k]`), and the inner level to the serialization of variables as defined in <<serialization-of_variables>>.
-The inner level does not exist for scalar variables.
+* `values` is a vector consisting of the <<serialization-of_variables,serialized>> values of these <<derivative,`derivatives`>.
```

Comment:
I think you deleted too much of the explanation: the combination of value references and order. Or is this explained elsewhere?

Reply:
@Andreas: You are right, I overlooked the "order", so this is a special case.